traits/traits_listener.py

```diff
@@ -208,9 +208,9 @@ class ListenerItem(ListenerBase):
     #: be deferred until the associated trait is first read or set?
     deferred = Bool(False)
 
-    #: Is this an 'any_trait' change listener, or does it create explicit
+    #: Is this an 'anytrait' change listener, or does it create explicit
     #: listeners for each individual trait?
-    is_any_trait = Bool(False)
+    is_anytrait = Bool(False)
```

Comment:
Similar concern here, but I think it's safe.

Reply:
Yep, I'm fairly sure this is safe: no-one outside Traits is using the traits parsing machinery.